pkgs/development/python-modules/pre-commit-hook-ensure-sops/default.nix

```
@@ -0,0 +1,33 @@
+{ lib, buildPythonPackage, applyPatches, fetchFromGitHub, ruamel-yaml }:
+
+buildPythonPackage rec {
+  pname = "pre-commit-hook-ensure-sops";
+  version = "1.1";
+
+  src = fetchFromGitHub {
+    owner = "yuvipanda";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-8sMmHNzmYwOmHYSWoZ4rKb/2lKziFmT6ux+s+chd/Do=";
```

Explicitly reference a tag. Use `hash` over `sha256`.
```suggestion
    rev = "refs/tags/v${version}";
    hash = "sha256-8sMmHNzmYwOmHYSWoZ4rKb/2lKziFmT6ux+s+chd/Do=";
```